### PR TITLE
4 login page elements

### LINKED ISSUE #4: login page elements
create components and add the dummy (non functioning) components to the login page

### PR DESCRIPTION
**Pull request by aidanContrastAI**
Closes #4

### CODE REVIEW

File: client/src/styles/home.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (SebBolh) [inline client/src/styles/home.css:1]: Combing the styles into one stylesheet called style.css

Comment (aidanContrastAI) [reply client/src/styles/home.css:1]: Combined!

File: client/src/components/login.js [hunk 2]
```diff
@@ -1,11 +1,26 @@
 import React from "react";
 import { Switch, Route, Link } from "react-router-dom";
 import "bootstrap/dist/css/bootstrap.min.css";
+import "../styles/login.css";
```
[the remark this answers is not in the document]
Comment (aidanContrastAI) [reply client/src/components/login.js:4]: Updated!

File: client/src/components/login.js [hunk 3]
```diff
@@ -1,11 +1,26 @@
...
 
 function Login() {
   return (
-    <div className="App">
-      Login.
+    <div class="full-screen-container">
```
[the remark this answers is not in the document]
Comment (aidanContrastAI) [reply client/src/components/login.js:8]: All background images removed

[the remark this answers is not in the document]
Comment (aidanContrastAI) [reply client/src/components/login.js:8]: incorporated grid system

File: client/src/styles/login.css [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+.login-box {
+  --color: hsl(var(--primary-light-hsl), 0.9);
+  background-color: var(--color);
+  box-shadow: 0 0 15px 0 var(--color);
+  height: 80vh;
+  border-radius: 5%;
+  padding: 40px 50px;
+  margin: 10%;
+  min-width: 500px;
+  max-width: 1000px;
```
[the remark this answers is not in the document]
Comment (aidanContrastAI) [reply client/src/styles/login.css:35]: removed unnecessary pixel values and used position absolute for children (also included min and max width/height for login-box responsiveness when window size is too small)

File: client/src/styles/login.css [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+.logo {
+  object-fit: contain;
+  margin: 0;
+  width: 100%;
+  height: 250px;
```
[the remark this answers is not in the document]
Comment (aidanContrastAI) [reply client/src/styles/login.css:42]: Makes sense! Got rid of strict pixel definitions.

File: client/src/components/login.js [hunk 6]
```diff
@@ -1,11 +1,26 @@
...
+      <div class="login-box">
```
[the remark this answers is not in the document]
Comment (aidanContrastAI) [reply client/src/components/login.js:9]: Minimized design, but I still like the shadows effect (gives a modern 3D look) as used by facebook login page. Can we keep this or is this a strong feeling?

File: client/src/components/login.js [hunk 7]
```diff
@@ -1,11 +1,26 @@
...
+        <img class="logo" src={require("../images/ContrastLogo.png")} alt="ContrastLogo"/>
+        <form class="form">
+          <div class="form-group">
+            <label for="username">Username</label>
```
[the remark this answers is not in the document]
Comment (aidanContrastAI) [reply client/src/components/login.js:13]: Deleted extra text!

File: client/src/components/login.js [hunk 8]
```diff
@@ -1,11 +1,26 @@
...
+            <input type="text" id="username" placeholder="Enter Username"/>
+          </div>
+
+          <div class="form-group">
+            <label for="password">Password</label>
```
[the remark this answers is not in the document]
Comment (aidanContrastAI) [reply client/src/components/login.js:18]: Deleted!

File: client/src/components/login.js [hunk 9]
```diff
@@ -1,11 +1,26 @@
...
   return (
-    <div className="App">
-      Login.
+    <div class="full-screen-container">
+      <div class="login-box">
+        <img class="logo" src={require("../images/ContrastLogo.png")} alt="ContrastLogo"/>
+        <form class="form">
+          <div class="form-group">
+            <label for="username">Username</label>
+            <input type="text" id="username" placeholder="Enter Username"/>
```
[the remark this answers is not in the document]
Comment (aidanContrastAI) [reply client/src/components/login.js:14]: Deleted!

File: client/src/components/login.js [hunk 10]
```diff
@@ -1,11 +1,26 @@
...
+          </div>
+
+          <div class="form-group">
+            <label for="password">Password</label>
+            <input type="password" id="password" placeholder="Enter Password" />
```
[the remark this answers is not in the document]
Comment (aidanContrastAI) [reply client/src/components/login.js:19]: Deleted! :)

**Pull request merged**